bizdemo/easy_note/note/errno/errno.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package errno
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/cloudwego/kitex-examples/bizdemo/easy_note/note/kitex_gen/kitex/demo/note"
+)
+
+type Errno struct {
+	Code int64
+	Msg  string
+}
+
+func (e Errno) Error() string {
+	return fmt.Sprintf("code=%d,msg=%s", e.Code, e.Msg)
+}
+
+var (
+	Success    = Errno{Code: 0, Msg: "success"}
+	ServiceErr = Errno{Code: 10001, Msg: "服务器开小差了"}
```

Comment:
Might want to consider to have an error message of both English and Chinese!

In this case
```go
var (
    ...
    ServiceErr = Errno{Code: 10001, Msg: "服务器开小差了 (Service is unable to start successfully)"}
    ParamErr = Errno{Code: 10002, Msg: "参数错误 (Wrong Parameter has been given"}
)
```
Something like that? I think it will help for English speaking contributors in the future:) I can help with the translation for the other errors as well if needed though my chinese has been really rusty.